doc/history.txt

```diff
@@ -53,6 +53,8 @@ we will remove the features in the next major release.
 
 Added/fixed:
 ============
+(*) CTL-895 Removing CurrentDispatcher property from IDispatcherService interface. Adding methods to IDispatcher 

```

Comment:
We use ordering in the history.txt, but I can change this one once this has been accepted.